process/block/baseProcess.go

```diff
@@ -1150,7 +1160,15 @@ func (bp *baseProcessor) getRootHashes(currHeader data.HeaderHandler, prevHeader
 	case state.UserAccountsState:
 		return currHeader.GetRootHash(), prevHeader.GetRootHash()
 	case state.PeerAccountsState:
-		return currHeader.GetValidatorStatsRootHash(), prevHeader.GetValidatorStatsRootHash()
+		currMetaHeader, ok := currHeader.(data.MetaHeaderHandler)
+		if !ok{
+			return []byte{}, []byte{}
+		}
+		prevMetaHeader, ok := prevHeader.(data.MetaHeaderHandler)
+ 		if !ok{
+ 			return []byte{}, []byte{}
+	    }
+		return currMetaHeader.GetValidatorStatsRootHash(), prevMetaHeader.GetValidatorStatsRootHash()
```

Comment:
why do we return here the same roothash ?

Reply:
it's not the same, first is the current block roothash, second is the prev block roothash